arangod/Graph/Providers/ClusterProvider.h

```diff
@@ -140,6 +140,7 @@ class ClusterProvider {
     Vertex _vertex;
     Edge _edge;
     bool _fetched;
+    size_t _localSchreierIndex = std::numeric_limits<size_t>::max();  // to be removed later
```

Comment:
Is this needed?
I do not see related .cpp code.
maybe this was made obsolete in the original.